pkg/webhook/mutation/pod/metadata/workload.go

```diff
@@ -58,6 +58,10 @@ func findRootOwnerOfPod(ctx context.Context, clt client.Client, pod *corev1.Pod,
 		return nil, errors.WithStack(err)
 	}
 
+	if podPartialMetadata.ObjectMeta.Name == "" {
+		log.Info("podName is empty")
+	}
```

Comment:
this will just spam the logs.

try it out with a `Deployment` that has 100 replicas
this line will appear 100 times, not the most helpful 😅

Reply:
also `findRootOwner` does not change `podPartialMetadata`, it just used as a starting point for the recursion

so printing it here is even more questionable IMO, because it would MAYBE be interesting before trying to find the owner.
- The "MAYBE" comes from the fact that it can happen that you have `Name` and `GeneratedName` set, because the pod was not `Created` but  `Updated`, so only looking at the `Name`/`GeneratedName` will give you an incomplete picture.

Reply:
Okay that makes sense of course, I think we could get rid off it anyways, now that we went for dealing with owners when they are no controller